linkml_model_enrichment/bioportal-enum-annotation.py

```diff
@@ -89,54 +117,63 @@ def replace(string, substitutions):
 
 
 @click.command()
```

Comment:
we may want to think about a yaml config file in the future

(the config file would be described using linkml)